code/app.py

```diff
@@ -26,29 +30,39 @@
 def static_file(path):
     return app.send_static_file(path)
 
-
-@app.route("/api/config", methods=["GET"])
+AZURE_AUTH_TYPE = os.environ.get("AZURE_AUTH_TYPE", "keys")
```

Comment:
The code base has had `black` and `flake8` run against it. If you do this, then we may see less merge conflicts